### ISSUE TITLE
relevance computation not performed

### ISSUE THREAD

**Issue by tanluDIMA** [maxdreyer/pcx#1]
First of all, thanks for the inspiring work. I have been following along from the early release of zennit to crp and now pcx. I am running tut0 with the following environmental setup (on Win11):
python 3.9.18
pytorch 1.12.1 cuda 11.6
zennit 0.4.6
zennit-crp 0.6.0

I noticed a rather strange behavior  in running 
`fv.run(composite, 0, len(dataset) // 1, batch_size=32).` 
Specifically, the LRP computations, which is to be realized via `basichook.backward` registered as hooks to input tensors to a module, was not performed. Consequently, the raw gradients were collected as relevance, next to the activations. A quick debugging shows that the hooks registered to the grad_fcn of tensors were never activated during the back propagation process 
`CondAttribution._attribute::self.backward(pred, grad_mask, exclude_parallel, cond_l_names, layer_out) `

However, the (pre-)forward hooks that were registered to modules are all functioning normally. The registration of the basichook.backward hooks, which is performed in the hook.pre_forward fucntion, also seems to be functioning. Only that the registered basichook.backward were never invoked during a backpropagation process. 

I am sure that there must be something that is missing on my side but I am currently having troubles to quickly identify the problems. I am running tut0 in a non-notebook format without changing the code (except from importing all packages at once at the  beginning of a python file). 

Any advices will be much appreciated!
